contrib/better-defaults/keybindings.el

```diff
@@ -14,3 +14,5 @@
 
 ;; emacs state bindings
 (define-key evil-emacs-state-map (kbd "C-o") 'evil-execute-in-normal-state)
+(add-hook 'dired-mode-hook (lambda ()
+                             (local-set-key (kbd "M-o") 'dired-display-file)))
```

Comment:
Why a local-set-key and a hook ? Can it be just set once and for all using a define-key ?